proto/poa/tx.proto

```
@@ -46,3 +48,14 @@ message MsgRemoveValidator {
 // MsgRemoveValidatorResponse defines the response for removing an existing
 // validator
 message MsgRemoveValidatorResponse {}
+
+// MsgSelfRemoveValidator defines a message for a validator to remove itself
+message MsgSelfRemoveValidator {
+  option (cosmos.msg.v1.signer) = "address";
```

Buf (1.70.0)

[error] cannot find `cosmos.msg.v1.signer` in this scope

(COMPILE)